teamsupport/services.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from demands import HTTPServiceClient
+from lxml import etree
+
+from teamsupport.utils import to_xml
+
+
+class XMLHTTPServiceClient(HTTPServiceClient):
+    def _format_xml_request(self, request_params):
+        data_set = request_params.get('data') is not None
+        if request_params.get('send_as_xml') and data_set:
+            data = request_params['data']
+            if not isinstance(data, etree._Element):
```

Comment:
Maybe this should check if it's a dict, rather than checking if it's not an XML element?

I think that makes what's going on slightly clearer. At first I thought this was turning XML strings into trees, and then back into strings.